usaon_vta_survey/templates/macros/forms/misc.j2

```diff
@@ -1,38 +1,38 @@
 {% macro observing_system_fields(form) -%}
```

Comment:
likely can rename this something like `response_object_fields` and use for all besides `Application`